satisfaction.js

```diff
@@ -11,7 +11,7 @@ function plugin_satisfaction_load_defaultvalue(root_doc, default_value){
     }
 
     $.ajax({
-        url: root_doc+'/plugins/satisfaction/ajax/satisfaction.php',
+        url: root_doc+'/ajax/satisfaction.php',
```

Comment:
Not sure about this one.

Reply:
Called from this line:
https://github.com/pluginsGLPI/satisfaction/blob/0eb5ff6475598088d01fc6150c211bbf561f0f95/inc/surveyquestion.class.php#L237